criu/files-reg.c

```diff
@@ -946,7 +947,7 @@ static int dump_ghost_remap(char *path, const struct stat *st, int lfd, u32 id,
 
 	pr_info("Dumping ghost file for fd %d id %#x\n", lfd, id);
 
-	if (st->st_size > opts.ghost_limit) {
+	if (st->st_blocks * ST_UNIT > opts.ghost_limit) {
 		pr_err("Can't dump ghost file %s of %" PRIu64 " size, increase limit\n", path, st->st_size);
```

Comment:
You've updated the check but error message is not updated, please do.

Reply:
Sure, I've done it.